src/reports/components/report-sections/report-footer.tsx

```diff
@@ -2,28 +2,16 @@
 // Licensed under the MIT License.
 import * as React from 'react';
 
-import { NewTabLink } from 'common/components/new-tab-link';
 import { NamedFC } from 'common/react/named-fc';
-import { toolName } from 'content/strings/application';
 import { SectionProps } from './report-section-factory';
 
-export type ReportFooterProps = Pick<SectionProps, 'environmentInfo'>;
+export type ReportFooterProps = Pick<SectionProps, 'environmentInfo'> & Pick<SectionProps, 'footerText'>;
```

Comment:
I think the syntax of `Pick` let you do this:
```typescript
Pick<SectionProps, 'environmentInfo' | 'footerText'>
```
I think this is less verbose

Reply:
Yes. Will change that.